docs/FAQ.md

```diff
@@ -0,0 +1,15 @@
+# FAQ
+
+## How to resolve merge conflicts in `pnpm-lock.yaml`?
+
+Do not try to pick either side of the lockfile.
+
+Drop both "ours" and "theirs" version (then the file will be in the base version).
```

Comment:
This is not clear, some devs will remove the lock file and recreate one.

```js
git checkout develop -- pnpm-lock.yaml
```

Tell them to run this command. (Replace `develop` to the base branch if it is not based on the `develop` branch)